webapp/src/components/cardDetail/cardDetailProperties.test.tsx

```diff
@@ -19,7 +19,7 @@ beforeEach(() => {
     FetchMock.fn.mockReset()
 })
 
-const wrapIntl = (children: any) => <IntlProvider locale='en'>{children}</IntlProvider>
+const wrapIntl = (children: ReactElement) => <IntlProvider locale='en'>{children}</IntlProvider>
```

Comment:
Hi @sbishel!

This function is copy-pasted in many unit tests. Also there is [one inside testUtils.tsx](https://github.com/mattermost/focalboard/blob/main/webapp/src/testUtils.tsx#L6).

So may be it can be imported and used instead of copy-pasting.

Reply:
Thanks @kamre. Great suggestion. Done...